app/controllers/api/v1/book_controller.rb

```diff
@@ -0,0 +1,20 @@
+module Api
+  module V1
+    class BookController < ApiController
+      def index
+        @books = Book.all
+        render_paginated @books, each_serializer: Api::V1::Serializer::BookSerializer
```

Comment:
you can use the Model here, not need to create a class variable if you do not use it outside the method

```ruby
render_paginated Book, each_serializer: Api::V1::Serializer::BookSerializer
```